### DISCUSSION TITLE
Orange Object (what is it?)

### DISCUSSION THREAD

**Discussion by beachspace** [Stellarium/stellarium#2611] [General]
Greetings, Can anyone tell me what this orange colored object is that is present on the Stellarium star map? Thanks
![orange object](https://user-images.githubusercontent.com/112538345/187605583-a89829b7-bdf5-46ae-ba45-917b86d6c211.jpg)

**Comment by Atque** [comment]
I think that’s an artifact in the DSS layer.

**Reply by beachspace** [reply]
Thanks! It has been floating about for some time now... :)

**Comment by BGvonB** [comment]
![Screenshot_20241104_180151](https://github.com/user-attachments/assets/56414d79-db05-444b-b55c-66f005421048)
Screenshot from my Samsung Galaxy A22 phone today... just watching Starlink objects criss-crossing my screen and... something odd appeared. 
I'm the previous comment about "DSS layer" means this fuzzy orange postage stamp among the stars is just some known graphics glitch and not a Vogon Destructor stationwd near Earth.
 :-)

**Reply by BGvonB** [reply]
Oops.
I'm HOPING the previous comment...

**Reply by gzotti** [reply]
Of course NASA is covering up something. Yes, Vogons!  :-)

Seriously, some image artifacts like this, or reflections from bright stars exist in the DSS layer. This may encourage you to observe right there and check yourself.

**Reply by BGvonB** [reply]
Stellarium is in the clear!

... or I've been neurolized,
![Screenshot_20241109_173722_DuckDuckGo](https://github.com/user-attachments/assets/b6baa276-5a17-4856-a354-50a032b1f651)

... and my cheapie Samsung mobile phone has unadvertised Vogon spotting capabilities: in the sky (with Stellarium) and on the ground (with Google Maps as per attached screen shot taken last week).
:-|
Was that lightning last night or a giant flashy thing near Ballarat?

**Reply by beachspace** [reply]
seems the orange blobs have arrived on planet earth.
